typescript/cli/src/check/warp.ts

```diff
@@ -70,3 +74,20 @@ export async function runWarpRouteCheck({
 
   logGreen(`No violations found`);
 }
+
+function verifyDecimalsAndScale(
```

Comment:
Instead of throwing can we have this report the decimals and which chain has an inconsistent setting just like lee did it for the contract verification? This way we can report all the violations instead of throwing early and missing any other possible mismatches